README.md

```diff
@@ -15,7 +15,10 @@ The plugin acts as an Azure Active Directory Application and must be configured
 
 In the Jenkins **Configure System** page, configure the following two options in the **Azure Key Vault Plugin** section
 * **Key Vault URL** - The url where your keyvault resides (e.g. `https://myvault.vault.azure.net/`)
-* **Credential ID** - The ID associated with a secret in the Jenkins secret store. Supported types are: **Microsoft Azure Service Principal** and **Managed Identities for Azure Resources**.
+* **Credential ID** - The ID associated with a secret in the Jenkins secret store. Supported types are: 
+    - **Microsoft Azure Service Principal**
+    - **Managed Identities for Azure Resources**
+    - **SystemAssigned Identities for Azure Resources**
```

Comment:
this isn't a credential type that I can see?
```suggestion
```

Reply:
You are right, both are of type ``Managed Identity`, one is `User assigned` and the other is `System assigned`.
I'll try to make this clearer.